src/rdb.c

```diff
@@ -1840,6 +1841,18 @@ static int _lpEntryValidation(unsigned char *p, unsigned int head_count, void *u
         }
     }
 
+    /* Validate TTL field. */
+    if (data->count % data->tuple_len == 2) {
+        long long expire_at;
+        /* Must be an integer. */
+        if (!lpGetIntegerValue(p, &expire_at)) return 0;
+        /* Must be less than EB_EXPIRE_TIME_MAX. */
+        if (expire_at < 0 || (uint64_t)expire_at > EB_EXPIRE_TIME_MAX) return 0;
+        /* The current TTL should not be less than the last one. */
+        if ((uint64_t)expire_at < data->last_expireat) return 0;
+        data->last_expireat = expire_at;
+    }
+
```

Comment:
this is a good idea, however it also bring a lot of changes, and all `lpValidateIntegrity*` methods will be changed.
i still want to leave it as now, and optimize it in the future if it doesn't meet the needs.
i started by simply copying a new lpexValidateIntegrity, then i chose to put the TTL fields validation in `_lpEntryValidation` because i thought it would be the least chage.